docs/DafnyRef/Topics.md

```diff
@@ -667,34 +665,33 @@ For anyone who may have felt that the intuitive proof felt too easy, here is a f
 proof using [the greatest forall definition](#eq-greatest-is-forall), which relies only on induction.  We massage the
 general form of our proof goal:
 
-<!--
-|~~~|~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~|
+
+|---|----------------------------------------------------------------------------------|
```

Comment:
This bit doesn't seem to render correctly for me (at least, viewing it from the GitHub interface at https://github.com/davidcok/dafny/blob/cok-docs-extreme/docs/DafnyRef/Topics.md).

Reply:
Fixed this and a few other formatting problems I saw